src/main/python/rlbot/agents/base_agent.py

```diff
@@ -48,11 +48,13 @@ def get_output_vector(self, game_tick_packet):
 
     def send_quick_chat(self, team_only, quick_chat):
         """
-        Sends a quick chat to the other bots.
+        Sends a quick chat to the other bots.  If it is None it does not send a quick chat to other bots
```

Comment:
This new sentence is a bit confusing. Are you talking about QuickChats.CHAT_NONE or python's None? Why would somebody pass either of those?

Reply:
This will probably mostly be used for debugging purposes.